blockly-ws-search/workspace_search.js

```diff
@@ -242,13 +276,15 @@ class WorkspaceSearch {
    */
   close() {
     this.setVisible(false);
+    this.workspace_.markFocused();
     this.clearBlocks();
     console.log("Close search bar");
   }
 
   /**
    * Shows or hides the workspace search bar.
-   * @param {boolean} show True to set the search bar as visible. False otherwise. 
+   * @param {boolean} show True to set the search bar as visible. False
+   *    otherwise.
```

Comment:
Optional suggestion: 
`{boolean} show Whether to set the search bar as visible.`